nautobot/core/views/renderers.py

```diff
@@ -245,7 +243,7 @@ def get_context(self, data, accepted_media_type, renderer_context):
             context["created_by"] = created_by
             context["last_updated_by"] = last_updated_by
             associated_contacts = instance.associated_contacts.restrict(request.user, "view").order_by("role__name")
```

Comment:
Should this be inside the `if` block below? We don't need to hit the DB here if we aren't actually going to use this data.

Reply:
More generally, why is the logic here so different from lines 145-158 above in `generic.py`?

Reply:
You are right, they should mirror each other